### DISCUSSION TITLE
Guidance Looking Forward

### DISCUSSION THREAD

**Discussion by slundberg** [guidance-ai/guidance#337] [General]
Hi all! Sorry to have been silent here for a while, this past month has obviously been a real challenge for Guidance maintenance (due to external circumstances). Our hope is to bring back a healthy pace of innovation soon, and @Harsha-Nori will be posting more about that. In the meantime I apologize for the delay and look forward to the exciting opportunities ahead for improved LLM control.

**Comment by raivatshah** [comment]
@slundberg thanks so much! I still want to double-check: for now, would it be advisable to use guidance for production apps? Asking from a continuity perspective!

**Reply by faizanahemad** [reply]
Same question here.

**Comment by faizanahemad** [comment]
Is this maintained or support is removed for any future works?
